### DISCUSSION TITLE
docker

### DISCUSSION THREAD

**Discussion by J09715** [lollipopkit/flutter_server_box#220] [General]
docker管理界面报错，更新最新版后出现的问- [![Screenshot_2023-11-20-22-27-29-261_tech.lolli.toolbox-edit.jpg](https://github.com/lollipopkit/flutter_server_box/assets/95364516/5474da64-225b-4c46-ab58-e32469d17e09)

**Comment by lollipopkit** [comment]
已于之前解决，如果仍然存在，请开启issue
